### DISCUSSION TITLE
Assign Specific Worker or Team

### DISCUSSION THREAD

**Discussion by Selbyl** [sdwebui-w-horde/sd-webui-stable-horde-worker#80] [Ideas]
So I have a few old mining rigs and I wanted to set up a SD platform for some friends to play with. Stable Horde is the easiest way tocombine all my resources, can this extension assign tasks to a specific API key so that only my hardware is being used?

**Comment by theUpsider** [comment]
I dont directly get what you mean. By saying "assign tasks to a specific key" to you mean that only certain API keys have certain models?

**Comment by MaikoTan** [comment]
If you're using [stable-ui](https://aqualxx.github.io/stable-ui/) as the webui for Stable Horde, you'll find there's an option called "Use Specific Worker" in the options page, does this solve your problems?
